go.mod

```diff
@@ -218,3 +227,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
```

Comment:
yeah, this looks weird, but for a good reason: [containerd: go mod friendly](https://github.com/containerd/containerd/issues/3031)

> Containerd currently causes issues for downstream projects using go mod. Since this is the official Go dependency management tool now, support would be nice.